Automattic/jetpack
# Post actions: Rename "Copy" to "Duplicate"

Related to https://github.com/Automattic/dotcom-forge/issues/5078.

## Proposed changes:

This diff renames the "Copy" post action to "Duplicate." Copy is not great semantically as the action the user really is taking is to duplicate an entry in the WordPress site.

In fact, it's misleading, as there is a notice in Calypso saying that this feature is actually meant to duplicate a page:

![image](https://github.com/Automattic/jetpack/assets/26530524/8b60baca-afda-40a8-84b2-365379cf531f)

So I propose to rename it. For starters, I just renamed the user-facing side, but if we want, I can also refactor the code so it says "duplicate" everywhere instead of "copy."

### Other information:

~- [ ] Have you written new tests for your changes, if applicable?~
~- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?~

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

In any site with the Jetpack plugin installed, verify that the duplicate post/page functionality says "Duplicate" instead of "Copy."

| Before | After |
| ------- | ------ |
| <img width="365" alt="image" src="https://github.com/Automattic/jetpack/assets/26530524/2beee5b4-bef5-47d4-82f6-0df3e2b35e3e"> | <img width="365" alt="image" src="https://github.com/Automattic/jetpack/assets/26530524/f6885d3f-ede0-4ac2-bb27-02e5dce42ba8"> |
